src/actions/course.js

```diff
@@ -32,6 +33,37 @@ export function createCourse(course) {
   }
 }
 
+/**
+ * Update a course's attributes
+ */
+const updateCourseRequest = makeActionCreator(
+  types.UPDATE_COURSE.REQUEST,
+  'courseId',
+  'attributes'
+)
+export const updateCourseSuccess = makeActionCreator(
```

Comment:
I believe there's currently no reducer to handle this case - that should be added so that the course is properly updated in the Redux store when the change has been accepted by the server.